### DISCUSSION TITLE
Curious about Obsidian tracker plugin

### DISCUSSION THREAD

**Discussion by rohanbatraio** [RafaelGB/obsidian-db-folder#465] [General]
Is there anyone who is using Db folder + obsidian tracker plugin together, If yes then i am curious how do you track YAML data in obsidian tracker. Would love to see your examples. I am facing a learning curve here.

**Comment by rohanbatraio** [comment]
this is the plugin specifically i am indicating about
https://github.com/pyrochlore/obsidian-tracker

**Comment by RafaelGB** [comment]
I recommend by far Obsidian Charts instead of tracker if you have knowledge of javascript.

This is my actual daily tracker of working hours. The day was as good as big is the dot

```js
//dataviewjs block
const { DateTime, Duration } = luxon
const dvData = dv.pages('"📔 Mi diario"');
const uniqueStatus = [...new Set(dvData.Status)];
const data = [];
dvData.forEach(d=>{
	if(d.RealWorkHours){
		const datetime = DateTime.fromFormat(d.file.name, 'yyyy-MM-dd');
		const daysFromNow = datetime.diffNow("days").toFormat("d")*(-1);
		let radius = 1;
		if(d.GoalAchieved){radius+=2}
		if(d.GoingOut){radius+=2}
		radius+=d.file.tasks.completed.length;
		const daily={
			x: daysFromNow,
			y: d.RealWorkHours.hours,
			r: radius
		};
		
		data.push(daily);
	}
});
const datasets = {
  datasets: [{
    label: 'Working Hours',
    data: data,
    backgroundColor: 'rgb(255, 99, 132)'
  }]
};
const chartData = {  
label: 'Working Hours',
	type: 'bubble',  
	data: datasets,
	 backgroundColor: 'rgb(255, 99, 132)',
	options: {}
}  
  
window.renderChart(chartData, this.container);
```

<img width="801" alt="Screenshot 2022-10-03 at 18 09 38" src="https://user-images.githubusercontent.com/11924043/193626272-38735e4b-d7ce-456f-8b59-b5d74d0b102e.png">

**Comment by rohanbatraio** [comment]
That looks awesome and just works kinda workflow, surely gonna try your approach.

**Reply by rohanbatraio** [reply]
I was hoping for a simple approach, as if i really want to pull up limits i can integrate python :rofl: and freaking use matplotlib or what not data science modules, that would be ridiculous but fun to try. But that would make me put in a lot of time. Moreover thank you for sharing the dataviewjs code snippet along with your reply. Would definitely save some time.

as always gonna checkout the documentation for knowing about Obsidian Charts. Lets see what this plugin gets me around.

**Comment by rohanbatraio** [comment]
I am curious, do you extract information out of obsidian? 
If yes then how?

**Reply by RafaelGB** [reply]
its all inside the vault. If you need information outside you can use templater scripts, even you can run python :)

**Reply by rohanbatraio** [reply]
Gotta do my research now, everyday i feel like my imposter syndrome is on peak. :laughing: Templater scripts + Python = ofcourse i am giving a try to this one.
